<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PZSculpt (Working title)</string>
  </property>
  <property name="styleSheet" >
   <string>QWidget{
	background-color: #9c978c;
	color: #fff;
	font-size: 12px;
}


QFrame{
	border: 1px solid #675f4b;
	border-radius: 5px;
}

QGroupBox{
	border: 1px solid #675f4b;
	border-radius: 5px;
	padding: 5px 0px;
	margin-top: 6px;
	font-weight: bold;
}
QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: center top;
	padding: 0px 10px;
} 


QPushButton{
	border-width: 1px;
	border-style: solid;
	border-color:  #534b3a;
	border-radius: 10px;
	height: 22px;
	min-height: 22px;
	max-height: 22px;
	padding: 0px 15px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	                                  stop: 0 #6f644b, stop: 1 #3f3828);
}
QPushButton:hover{
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
	                                  stop: 0 #6f644b,
                                      stop: 0.5 #6f644b,
                                      stop: 1 #3f3828);
}
QPushButton:pressed{
	background-color: #534b3a;
}


QMenuBar{
	background-color: #7d796f;
}
QMenuBar::item{
	background-color: #7d796f;
}
QMenuBar::item:selected{
	background-color: #8f8f8f;
}


QMenu {
	background-color: #7d796f;
	border: 1px solid #8f8f8f;
}
QMenu::item {
	padding: 0px 30px 0px 10px;
	background-color: transparent;
}
QMenu::item:selected{
	background-color: #8f8f8f;
}
QMenu::item:disabled{
	color: #aaa;
}
QMenu::separator{
	height: 1px;
	margin: 2px 5px;
	background-color: #534b3a;
}


QSpinBox{
	background-color: #e9e9e9;
	color: #000;
	border: 1px solid #000;
	border-radius: 2px;
	padding-right: 20px;
}
QSpinBox::up-button {
	subcontrol-position: top right;
	width: 18px;
	border-image: url(:/images/res/spinboxup.png) 1;
}
QSpinBox::down-button {
	subcontrol-position: bottom right;
	width: 18px;
	border-image: url(:/images/res/spinboxdown.png) 1;
}
QSpinBox::up-arrow {
	image: url(:/images/res/arrowup.png);
	width: 5px;
	height: 5px;
}
QSpinBox::down-arrow {
	image: url(:/images/res/arrowdown.png);
	width: 5px;
	height: 5px;
}
QSpinBox::up-arrow:pressed{
	image: url(:/images/res/arrowup_pressed.png);
}
QSpinBox::down-arrow:pressed{
	image: url(:/images/res/arrowdown_pressed.png);
}


QLabel{
	background-color: #9c927b;
	height: 15px;
	min-height: 15px;
	max-height: 15px;
	padding: 0px 10px;
	border-radius: 0px;
}


QSlider::groove:horizontal {
	border: 1px solid #000;
	height: 2px;
	background-color: #fff;
}
QSlider::handle:horizontal {
	border: 1px solid #000;
	border-radius: 4px;
	height: 5px;
	width: 20px;
	background-color: #86734e;
	margin: -5px 0px;
}


QRadioButton::indicator {
	width: 15px;
	height: 15px;
}
QRadioButton::indicator::unchecked {
	image: url(:/images/res/radiobutton_unchecked.png);
}
QRadioButton::indicator:unchecked:pressed {
	image: url(:/images/res/radiobutton_unchecked_pressed.png);
}
QRadioButton::indicator::checked {
	image: url(:/images/res/radiobutton_checked.png);
}
QRadioButton::indicator:checked:pressed {
	image: url(:/images/res/radiobutton_checked_pressed.png);
} 


QCheckBox::indicator {
	width: 15px;
	height: 15px;
}
QCheckBox::indicator::unchecked {
	image: url(:/images/res/checkbox_unchecked.png);
}
QCheckBox::indicator:unchecked:pressed {
	image: url(:/images/res/checkbox_unchecked_pressed.png);
}
QCheckBox::indicator::checked {
	image: url(:/images/res/checkbox_checked.png);
}
QCheckBox::indicator:checked:pressed {
	image: url(:/images/res/checkbox_checked_pressed.png);
} 

QAction{
	background-color: #f00;
}

QTextEdit{
	padding: 20px 10px;
	max-height: 60px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="2" >
     <widget class="QFrame" name="frameTop" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBoxBrush" >
         <property name="title" >
          <string>Brush</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QRadioButton" name="radioButtonDraw" >
            <property name="text" >
             <string>&amp;Draw</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonSmooth" >
            <property name="text" >
             <string>&amp;Smooth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonPinch" >
            <property name="text" >
             <string>&amp;Pinch</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="labelSize" >
           <property name="text" >
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="spinBoxSize" >
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QSlider" name="horizontalSliderSize" >
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="labelIntencity" >
           <property name="text" >
            <string>Intencity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="spinBoxIntencity" >
           <property name="value" >
            <number>50</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QSlider" name="horizontalSliderIntencity" >
           <property name="value" >
            <number>50</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="labelStrength" >
           <property name="text" >
            <string>Strength</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QSpinBox" name="spinBoxStrength" >
           <property name="value" >
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QSlider" name="horizontalSliderStrength" >
           <property name="value" >
            <number>25</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="PZSculpt::GLWidget" native="1" name="canvas" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QFrame" name="frameRight" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="labelGeometry" >
         <property name="text" >
          <string>Geometry</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="pushButtonDown" >
           <property name="text" >
            <string>down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonUp" >
           <property name="text" >
            <string>up</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSubd" >
         <property name="text" >
          <string>subdivide</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelMeshInfo" >
         <property name="text" >
          <string>Mesh Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditMeshInfo" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>281</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport" />
    <addaction name="actionExport" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Edit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Edit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionImport" >
   <property name="text" >
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionExport" >
   <property name="text" >
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PZSculpt::GLWidget</class>
   <extends>QWidget</extends>
   <header location="global" >glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButtonDraw</tabstop>
  <tabstop>radioButtonSmooth</tabstop>
  <tabstop>radioButtonPinch</tabstop>
  <tabstop>spinBoxSize</tabstop>
  <tabstop>horizontalSliderSize</tabstop>
  <tabstop>spinBoxStrength</tabstop>
  <tabstop>horizontalSliderStrength</tabstop>
  <tabstop>pushButtonDown</tabstop>
  <tabstop>pushButtonUp</tabstop>
  <tabstop>pushButtonSubd</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>257</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>594</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel" >
     <x>408</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderStrength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxStrength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>594</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel" >
     <x>408</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>408</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel" >
     <x>594</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxStrength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderStrength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>408</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel" >
     <x>594</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxIntencity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderIntencity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>385</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>464</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderIntencity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxIntencity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>568</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>395</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
